backend/src/cusco/main.py

```
@@ -38,16 +50,34 @@
+    app.state.bg_tasks = bg_tasks
+    try:
+        yield
+    finally:
+        logger.info("Cusco shutting down.")
+        for t in bg_tasks:
+            t.cancel()
+        for t in bg_tasks:
+            try:
+                await t
+            except (asyncio.CancelledError, Exception):
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)